packages/deployments/contracts/contracts/core/connext/facets/InboxFacet.sol

```diff
@@ -99,7 +99,7 @@ contract InboxFacet is BaseConnextFacet {
    * @param _router The address the message is coming from.
    */
   modifier onlyRemoteRouter(uint32 _origin, bytes32 _router) {
-    if (!_isRemoteRouter(_origin, _router)) {
+    if (!_isRemoteHandler(_origin, _router)) {
```

Comment:
Would be consistent to rename `onlyRemoteRouter` to something like `onlyRemoteXAppHandler`